block/manager.go

```diff
@@ -414,6 +423,10 @@ func (m *Manager) freezeNode(err error) {
 		return
 	}
 	uevent.MustPublish(m.Ctx, m.Pubsub, &events.DataHealthStatus{Error: err}, events.HealthStatusList)
+	err = m.DAClient.Stop()
```

Comment:
why it's here? the DACLient started outside of the Manager. not sure he's supposed to shut it down.
DA will be stopped when **Node** is stopped
same with SL client